astropy/io/fits/fitstime.py

```diff
@@ -0,0 +1,399 @@
+    global_info : dict
+        Global time reference frame information.
+    """
+
+    # Translate FITS deprecated scale into astropy scale, or else just convert
+    # to lower case for further checks.
+    global_info['TIMESYS'] = FITS_DEPRECATED_SCALES.get(global_info['TIMESYS'],
+                                                        global_info['TIMESYS'].lower())
+
+    # Verify global time scale
+    if not global_info['TIMESYS'] in Time.SCALES:
+
+        if global_info['TIMESYS'] == 'gps':
+            warnings.warn(
+                'Global time scale (TIMESYS) has a FITS recognized time scale '
+                'value "GPS". In Astropy, "GPS" is a time from epoch format '
+                'which runs synchronously with TAI; GPS is approximately 19 s '
+                'ahead of TAI. Hence, this format will be used.', AstropyUserWarning)
+            global_info['FORMAT'] = 'gps'
+            global_info['TIMESYS'] = 'tai'
+
+        if global_info['TIMESYS'] == 'local':
+            warnings.warn(
+                'Global time scale (TIMESYS) has a FITS recognized time scale '
+                'value "LOCAL". However, the standard states that "LOCAL" should be '
+                'tied to one of the existing scales because it is intrinsically '
+                'unreliable and/or ill-defined. Astropy will thus use the default '
+                'global time scale "UTC" instead of "LOCAL".', AstropyUserWarning)
+            global_info['TIMESYS'] = 'utc'
+
+        else:
+            raise AssertionError(
+                'Global time scale (TIMESYS) should have a FITS recognized '
+                'time scale value (got {!r}). The FITS standard states that '
+                'the use of local time scales should be restricted to alternate '
+                'coordinates.'.format(global_info['TIMESYS']))
+
+
+def _verify_column_info(column_info, global_info):
+    """
+    Given the column-specific time reference frame information, verify that
+    each column-specific time coordinate attribute has a valid value.
+    Return True if the coordinate column is time, or else return False.
+
+    Parameters
+    ----------
+    global_info : dict
+        Global time reference frame information.
+    column_info : dict
+        Column-specific time reference frame override information.
+    """
+
+    scale = column_info.get('TCTYP', None)
+    unit = column_info.get('TCUNI', None)
+    pos = column_info.get('TRPOS', None)
+
+    if pos is not None:
+
+        if pos == 'TOPOCENTER':
+            column_info['TRPOS'] = EarthLocation(global_info['OBSGEO-X'],
+                                                 global_info['OBSGEO-Y'],
+                                                 global_info['OBSGEO-Z'],
+                                                 unit='m')
+        else:
+            column_info['TRPOS'] = None
+
+    if scale is not None:
+
+        if scale.lower() in Time.SCALES:
+            column_info['TCTYP'] = scale.lower()
```

Comment:
I think I would do the same here, replace `TCTYP` with `scale` (and `FORMAT` with `format` below).